### DISCUSSION TITLE
Errors in logs: record "new" has no field "raw_user_meta_data"

### DISCUSSION THREAD

**Discussion by organicnz** [supabase/supabase#12284] [Questions] [answered]
Hi guys,

Could you please help with this error `record "new" has no field "raw_user_meta_data"`? Not sure where it comes from yet :) 

Thank you :)

![image](https://user-images.githubusercontent.com/4765090/217742828-ad1138a2-6d94-4e52-a357-f2890484fe1d.png)

**Comment by GaryAustin1** [comment, marked as answer]
It would be from a trigger function you have added most likely.  It is like you have a tigger function that was designed for auth.users and is being triggered by delete or is on another table that does not have that column.

**Reply by organicnz** [reply]
Thanks so much mate :)

I assume that these errors are also caused by the triger function :))

`relation "vault.secrets" does not exist`
`column "name" does not exist`
